### PR TITLE
Add quick-start into 'Testing and Deployment' section

### PR DESCRIPTION
**Pull request by lonng**
Signed-off-by: Lonng <heng@lonng.org>

### What problem does this PR solve?

Add quick-start into the 'Testing and Deployment' section, and change the recommendation deployment from Ansible to TiUP.

### Release note

- No need

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,25 +77,35 @@ When a node starts, the metadata of the Node, Store and Region are recorded into
...
+    ```
 
-For the production environment, use [Ansible](https://github.com/pingcap/tidb-ansible) to deploy the cluster.
+See [TiDB Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/) for other quick start methods.
+
+### Production deployment
 
-- [Deploy TiDB Using Ansible](https://github.com/pingcap/docs/blob/master/how-to/deploy/orchestrated/ansible.md)
-- [Deploy TiKV separately Using Ansible](https://tikv.org/docs/3.0/tasks/deploy/ansible/)
+For the production environment, use [TiUP](https://pingcap.com/docs/stable/how-to/deploy/orchestrated/tiup/) to deploy the cluster.
```
Comment (siddontang) [inline README.md:108]: how to deploy TiKV directly using TiUP?

File: README.md [hunk 2]
```diff
@@ -77,25 +77,35 @@ When a node starts, the metadata of the Node, Store and Region are recorded into
...
 
-- Try TiKV separately
-    - [Deploy TiKV Using Docker Compose/Swarm](https://tikv.org/docs/3.0/tasks/deploy/docker-compose/): To quickly test TiKV separately without TiDB using [`tidb-docker-compose`](https://github.com/pingcap/tidb-docker-compose/) on a single machine
-    - [Deploy TiKV Using Docker](https://tikv.org/docs/3.0/tasks/deploy/docker/): To deploy a multi-node TiKV testing cluster using Docker
-    - [Deploy TiKV Using Binary Files](https://tikv.org/docs/3.0/tasks/deploy/binary/): To deploy a TiKV cluster using binary files on a single node or on multiple nodes
+    ```shell script
+    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh 
+    ```
+  
+- Run a TiDB cluster (the latest stable version)
```
Comment (BusyJay) [inline README.md:92]: How to run TiKV without TiDB?

File: README.md (hunk 1 above)
Comment (lonng) [reply README.md:108]: The topology doesn't contain TiDB instances.

**Pull request closed without merging**